snmp/assets/dashboards/datadog_ndm_environment.json

```diff
@@ -0,0 +1,2760 @@
+                                            "query": "sum:datadog.netflow.aggregator.hash_collisions{$Agent_Hostname,$Custom_Tag} by {host}.as_count()"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "style": {
+                                        "line_type": "solid",
+                                        "line_width": "normal",
+                                        "palette": "purple"
+                                    }
+                                }
+                            ],
+                            "show_legend": true,
+                            "title": "Hash collisions",
+                            "title_align": "left",
+                            "title_size": "16",
+                            "type": "timeseries"
+                        },
+                        "id": 6036396658807928,
+                        "layout": {
+                            "height": 2,
+                            "width": 4,
+                            "x": 8,
+                            "y": 13
+                        }
+                    },
+                    {
+                        "definition": {
+                            "legend_columns": [
+                                "avg",
+                                "min",
+                                "max",
+                                "value",
+                                "sum"
+                            ],
+                            "legend_layout": "auto",
+                            "requests": [
+                                {
+                                    "display_type": "bars",
+                                    "formulas": [
+                                        {
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "queries": [
+                                        {
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "query": "sum:datadog.netflow.processor.flowsets{$Agent_Hostname,$Custom_Tag} by {host}.as_count()"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "style": {
+                                        "line_type": "solid",
+                                        "line_width": "normal",
+                                        "palette": "dog_classic"
+                                    }
+                                }
+                            ],
+                            "show_legend": true,
+                            "title": "NetFlows FlowSets",
```

Comment:
Note that the metric represent the netflow UDP packets received. But it's currently available only for netflow: https://github.com/DataDog/datadog-agent/blob/c364b135bd488256b1b760374a4d3d7106a93330/pkg/netflow/goflowlib/metric.go#L76-L86

Beside being used in NetFlow packet drop calculation, I'm wondering if we need it here. I think we can probably not have it for now and add it later if needed. WDYT?

Reply:
For NetFlow5, the metric represent the # of flows.

For NetFlow 9, it represent the number of flows but also NetFlow Templates.